File: back-python/1-basic/1-syntax/4.py
# [메소드 만들기]






# 파라미터가 없는 메소드
def a_print():
    print('a')

def b_print():
    print('b')

a_print()
b_print()






# 파라미터가 있는 메소드1
def add(a, b):
   result = a + b
   print("{} = {} + {}".format(result, a, b)) # 프린트에 포맷 다는 법!
add(1, 3)
add(100, 400)

def add10(a):
    d = a + 10
    print(d)
add10(3)
add10(10)




# 파라미터가 있는 메소드2
def banolim(a):
    result = round(a)
    print(result)

banolim(2.2)
banolim(2.7)




# 파라미터가 있는 메소드3
def banolim2(a, b):
    result1 = round(a)
    result2 = round(b)
    print(result1)
    print(result2)

banolim2(2.2, 2.7)

File: start/8-workspace/1-jung/5-js/1-js초급/4-반복/4.js
const a = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23]

// for (초기값; 실행조건; 한 루프를 실행한 뒤 실행)
// for (let i = 0; i < a.length; i++) {
//   console.log(a[i])
// }

// 3의 배수만을 출력하시오.
// for (let i = 0; i < a.length; i++) {
//   if (a[i] % 3 === 0) {
//     console.log(a[i])
//   }
// }

// 2의 배수이면서 3의 배수인 것을 출력하시오. 
// for (let i = 0; i < a.length; i++) {
//   if (a[i] % 2 === 0 && a[i] % 3 === 0) {
//     console.log(a[i])
//   }
// }

// 3의 배수이면서 4의 배수인 것을 출력하시오.
// for (let i = 0; i < a.length; i++) {
//   if (a[i] % 3 === 0 && a[i] % 4 === 0) {
//     console.log(a[i])
//   }
// }

// 3의 배수이거나 5의 배수인 것을 출력하시오. 
// for (let i = 0; i < a.length; i++) {
//   if (a[i] % 3 === 0 || a[i] % 5 === 0) {
//     console.log(a[i])
//   }
// }

// a 배열에서, 10이상의 수만 출력하시오.
// for (let i = 0; i < a.length; i++) {
//   if (a[i] >= 10) {
//     console.log(a[i])
//   }
// }

// 구구단 (이중포문)
// for (let i = 2; i < 10; i++) {
//   console.log(i, '단 시작 >>>')
//   for (let j = 1; j < 10; j++) {
//     console.log(i, 'x', j, '=', i * j)
//   }
// }

// 1에서 100까지의 홀수의 합
let result = 0
for (let i = 1; i < 101; i++) {
  if (i % 2 === 1) {
    result = result + i
  }
}
console.log(result)
